### ISSUE TITLE
Increase contrast between background and foreground

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#83]
Success Criteria 1.4.3 in WCAG 2 [requires a minimum of color contrast](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=lighthouse). The accessibility tests of  the Chrome developer tool Lighthouse suggests our contrast is too low.

Failing elements on the front page:
```HTML
<a class="navbar-brand" href="/"> Trix </a>
<a href="/login?next=/" class="btn btn-warning"> <span class="fa fa-sign-in"></span> Logg inn </a>
<a href="http://trix2.readthedocs.org/en/latest/student/gettingstarted.html">les kom igang guiden</a>
<p class="text-muted">Innføring i beregningsmodeller for biovitenskap</p>
<p class="text-muted">Introduksjon til objektorientert programmering (Python)</p>
<p class="text-muted">Objektorientert programmering (ny emnekode fra vår 2018)</p>
<p class="text-muted">Grunnkurs i objektorientert programmering. </p>
<p class="text-muted">Grunnkurs i objektorientert programmering (med Python)</p>
<p class="text-muted">Objektorientert programmering</p>
<p class="text-muted">Trening på Java for studenter fra INF1001 og INF1100 (Python-emner) før start på INF1010</p>
<p class="text-muted">Test</p>
```
